spatial/r3/mat_test.go

```diff
@@ -230,3 +230,59 @@ func BenchmarkQuat(b *testing.B) {
 		}
 	}
 }
+
+func TestMatHessian(t *testing.T) {
+	const (
+		tol = 1e-3
```

Comment:
In fact, with `h = 8e-4` I get the test to pass with a more stringent tolerance `tol = 5e-6`. However, that's quite tight and could cause flakiness in the test, so using `tol = 1e-5` is enough in my opinion.

Reply:
@vladimir-ch good find. added!